packages/core/src/App/Containers/Layout/footer.jsx

```diff
@@ -1,17 +1,30 @@
 import React from 'react';
-import { routes } from '@deriv/shared';
-import DefaultFooter from './default-footer';
-import TradingHubFooter from './trading-hub-footer';
 import { useLocation } from 'react-router-dom';
+import { useDevice } from '@deriv-com/ui';
+import { routes, isDisabledLandscapeBlockerRoute } from '@deriv/shared';
+import { DesktopWrapper } from '@deriv/components';
+import TradingHubFooter from './trading-hub-footer';
 
 const Footer = () => {
+    const { isDesktop, isTablet } = useDevice();
     const { pathname } = useLocation();
-    if (pathname !== routes.onboarding) {
-        return <TradingHubFooter />;
-    } else if (pathname === routes.onboarding) {
+    const is_hidden_landscape_blocker = isDisabledLandscapeBlockerRoute(pathname);
+    const show_in_tablet_routes = [routes.trade, routes.contract];
+
+    if (pathname === routes.onboarding) {
         return null;
-    }
-    return <DefaultFooter />;
+    } 
+        if (is_hidden_landscape_blocker) {
+            if (isDesktop) return <TradingHubFooter />;
+            if (isTablet && show_in_tablet_routes.includes(pathname)) return <TradingHubFooter />;
+            return null;
+        }
+        return (
+            <DesktopWrapper>
```

Comment:
Are we still using desktopwrapper?

Reply:
@henry-deriv Yes, we need it, because this footer used for other packages, for accounts or for cashier, for example, and there we still use old breakpoints